tests/deepspeed/test_deepspeed.py

```diff
@@ -165,7 +167,9 @@ def tearDown(self):
         super().tearDown()
 
         # reset the ds config global so that tests state doesn't leak
-        unset_hf_deepspeed_config()
+        with mockenv_context(**self.dist_env_1_gpu):
+            accelerator = Accelerator()
+            accelerator.state.initialized = False
```

Comment:
I don't think this changes the fast `accelerator.state` keeps the deepspeed config here. You should change `AcceleratorState` directly.